src/test/com/kainos/ea/service/JobsRequestServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package com.kainos.ea.service;
+
+import com.kainos.ea.dao.JobsDao;
+import com.kainos.ea.exception.DatabaseConnectionException;
+import com.kainos.ea.models.Job;
+import com.kainos.ea.util.DatabaseConnector;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class JobsRequestServiceTest {
+
+    JobsDao jobsDao = Mockito.mock(JobsDao.class);
+    DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+
+    JobsRequestService jobsRequestService = new JobsRequestService(jobsDao, databaseConnector);
+
+    Connection conn;
+
+    /*
+    Unit test 1
```

Comment:
This documentation isn't necessary. Overtime, it would become outdated. The test's name in the signature should tell you the scenario.